attestation/jsonattestor/json.go

```diff
@@ -0,0 +1,213 @@
+package jsonattestor
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/cyberphone/json-canonicalization/go/src/webpki.org/jsoncanonicalizer"
```

Comment:
nit: this is a pretty old module that doesn't seem to have been updated for a long while. At the same time though, the repo seems to still be active along with its maintainer.

Reply:
ended up bringing the code internal.  It should never change